src/ReportPortal.SpecFlowPlugin/ReportPortalHooks.cs

```diff
@@ -180,13 +180,19 @@ public void AfterScenario()
                     case ScenarioExecutionStatus.TestError:
                         status = Status.Failed;
 
+                        issue = new Issue
```

Comment:
Don't report issue, AA will not work.

Reply:
I tested it and AA works correctly (RP 4.2.0)